pages/integrations/output_framework.rst

```diff
@@ -40,6 +40,10 @@ the external receiver. This allows the Output Framework to keep receiving and re
 queuing messages, even if the external receiver is temporarily unavailable due to
 network issues. The journal has many configuration options which will be described below.
 
+By default, journal data for Framework Outputs will be stored in the same directory used
+for the Input journal.  This directory is controlled by the ``message_journal_dir`` value
```

Comment:
Thanks for the update here.  One more question on it, will there be a specific name for this journal? If so, we could note it, so they could keep track of it.

Reply:
I think the journal name is randomly chosen but specific to the input.